CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## x.x.x
+* Add support for adaptive banners. For more info, check out our [docs](https://dash.applovin.com/documentation/mediation/react-native/ad-formats/banners#adaptive-banners).
```

Comment:
The plugin technically already have support for adaptive banners, right? If using programmatic implementation.

Reply:
Right, so updated.